src/stats/signals.py

```diff
@@ -1,68 +1,61 @@
+from django.core.cache import caches
```

Comment:
You should be using the default cache here:
```suggestion
from django.core.cache import cache
```